### ISSUE TITLE
429 Client Error: Has there been a change in the rate limit?

### ISSUE THREAD

**Issue by rwsjsed** [hitbtc-com/hitbtc-api#148]
I have a program that has been downloading the orderbooks for a handful of symbols at less than 1000 requests per minute. This was stable for several weeks.  Recently I started getting a flood of 429 errors:

GET 429 Client Error: Too Many Requests for url: https://api.hitbtc.com/api/2

The errors usually come in batches and then stop for a period of time before resuming.  I have tried lowering the rate at which I request the orderbooks but this does not seem to change the result.  I have looked in the response header to see if there was a prescribed rate limit that I should adhere to but did not find one.

Is there a published rate limit?  Did it change recently? Is this behaving as expected?

**Comment by Wo0nder** [comment]
Bump. The same error when updating the price of the lot. I would like information about the limits.

**Comment by kz0** [comment]
Been having the same problem for almost a week. If the rate limit has changed please update the websocket documentation so users can switch.

**Comment by Legogris** [comment]
https://api.hitbtc.com/#rate-limiting
```
The Hit API Rate Limits apply since 22 March 2018 to determine the number of requests that can be made to the Hit API.

Rate Limits apply: * For the Market data, the limit is 100 requests/second per IP; * For Trading, the limit is 100 request/second per user; * For other requests, including Trading history, the limit is 10 requests/second per user.
```

I'm doing well under these limits and still get rate limited for a majority of requests. Even 20 per minute per IP ends up in heavy rate limiting.

**Comment by hitbtc-com** [comment]
@rwsjsed ,  this issue has already been fixed. Please check it and tell us if the problem still exists

**Comment by apollo4040** [comment]
@hitbtc-com Yes. Problem still exists. When I'm trying to create socket to get reports of my account I get HTTP/1.1 429 Too Many Requests.
